### ISSUE TITLE
calculate training error/accuracy percentage

### ISSUE THREAD

**Issue by johnneuland** [Tom-Alexander/regression-js#115]
Is it possible to somehow also obtain some sort of training error or answer accuracy percentage after performing polynomial linear regression? Any guide would help big time!

**Comment by JimLynchCodes** [comment]
@johnneuland, yes- this is the "r2" field of the object resulting from one of the regression functions (linear, exponential, etc.)
